src/chapters/booleans-and-conditionals/logical-operators.rst

```diff
@@ -0,0 +1,273 @@
+Logical Operators
+=================
+
+.. index:: operators
+
+Recall that an operator is one or more characters that carries out an action on its operand(s). In :ref:`data-and-variables` we learned about three types of operators:
+
+- Arithmetic operators, such as ``+``, ``-``, and ``%``.
+- The string operator ``+``.
+- Compound assignment operators, such as ``+=`` and ``-=``.
+
+Arithmetic and string operators take number and string operands, respectively, returning values of the same type. Compound assignment operators work similarly with numbers or strings, while also reassigning the value of the first, variable operand.
+
+Boolean Operators
+-----------------
+
+.. index::
+   single: operator; comparison
+   single: operator; boolean
+
+This chapter has introduced an additional type of operator, comparison operators, which include ``===``, ``<``, and others. These operators are part of a larger class known as **boolean operators**, so-called because they return a boolean value.
+
+There are three additional boolean operators that allow us to create more complex boolean expressions, and which we will learn about now.
+
+Logical AND
+^^^^^^^^^^^
+
+.. index:: ! &&
+
+.. index::
+   single: operator; boolean
+
+JavaScript allows us to create a compound boolean expression using the logical AND operator, ``&&``. A compound boolean expression is a boolean expression built out of smaller boolean expressions. 
+
+The operator takes two operands, and the resulting expression is ``true`` if *both* operands are ``true`` individually. If either operand is ``false``, the overall expression is ``false``.
+
+In English, the ``&&`` operator mirrors the use of the word "and" (hence the name "logical AND"). A sentence like "Roses are red and violets are blue," is true as a whole precisely because roses are actually red, and violets are actually blue. On the other hand, the sentence "Roses are red and violets are green," is false as a whole. While roses are indeed red, violets are not green. 
+
+Let's see how this works in code.
+
+.. admonition:: Example
+
+   .. sourcecode:: js
+      :linenos:
+
+      console.log(7 > 5 && 5 > 3);
+      console.log(7 > 5 && 2 > 3);
+      console.log(2 > 3 && 'dog' === 'cat');
+
+   **Output**
+
+   ::
+
+      true
+      false
+      false
+
+In line 1, ``7 > 5 && 5 > 3`` returns ``true`` because both ``7 > 5`` and ``5 > 3`` are ``true`` individually.
+
+In line 2, ``7 > 5 && 2 > 3`` returns ``false`` because one of the two expressions, ``2 > 3``, is ``false``.
+
+Like line 2, line 3 returns ``false`` because both sub-expressions are ``false``. Notice in this example that we can mix and match data types however we like, as long as both sides of the ``&&`` expression are themselves boolean expressions.
```

Comment:
Phrasing: Remove "Like line 2".  Also, remove "in this example", since it implies that there may be a case where we CANNOT mix and match data types.